ml/test.py

```diff
@@ -0,0 +1,102 @@
+import numpy as np
+import pandas as pd
+import torch
+import json
+import os
+
+from Modali import infer
+
+##################################
+#
+# In Dehatebert the labels are:
+#
+#   0: non-hate
+#   1: hate
+#
+# In dataset.json the labels are:
+#
+#   True:  toxic
+#   False: non-toxic
+#   /:     can't say
+#
+#################################
+
+toxic_id = 1
+non_toxic_id = 0
+
+# CONVERT DATASET LABELS INTO NEURALY LABELS
+df = pd.read_json("dataset.json")
+texts = df.loc[:,['text']]
+sentiments = df.loc[:,['label']]
+
+features = texts.to_numpy()
+labels = sentiments.to_numpy()
+
+dataset = []
+toxic = 0
+non_toxic = 0
+
+for j in range(0, len(features)):
+    if labels[j] == True: # Toxic
+        dataset.append({"text": str(features[j]), "label": toxic_id})
+        toxic += 1
+
+    elif labels[j] == False: # Non-Toxic
+        dataset.append({"text": str(features[j]), "label": non_toxic_id})
+        non_toxic += 1
```

Comment:
Just a bit more pythonic:
```suggestion
for label, feature in zip(labels, features):
    if label is True:  # Toxic
        dataset.append({"text": str(feature), "label": toxic_id})
        toxic += 1

    elif label is False:  # Non-Toxic
        dataset.append({"text": str(feature), "label": non_toxic_id})
        non_toxic += 1
```

Reply:
Generates empty list.